charts/tm-forum-api/values.yaml

```diff
@@ -294,4 +303,20 @@ apis:
     image: tmforum-service-catalog
     basePath: /tmf-api/serviceCatalogManagement/v4
 
```

Comment:
Please add documentation to the fields. See the format of the other property-documentation, its important for the doc generation.

Reply:
done